scripts/gha/print_matrix_configuration.py

```diff
@@ -79,11 +79,11 @@
 # To list avaliable packages, install u3d, and use cmd "u3d available -u $unity_version -p"
 # The packages below is valid only if Unity Hub is not installed.
 UNITY_PACKAGES = {
-  "2020.3.19f1": {"Default": ["Unity"], "Android": ["Android"], "iOS": ["Ios"], "Windows": ["Windows-mono"], "macOS": None, "Linux": ["Linux-mono"]},
-  "2019.4.30f1": {"Default": ["Unity"], "Android": ["Android"], "iOS": ["Ios"], "Windows": ["Windows-mono"], "macOS": None, "Linux": ["Linux-mono"]},
-  "2019.4.32f1": {"Default": ["Unity"], "Android": ["Android"], "iOS": ["Ios"], "Windows": ["Windows-mono"], "macOS": None, "Linux": ["linux-il2cpp"]},
-  "2018.4.36f1": {"Default": ["Unity"], "Android": ["Android"], "iOS": ["Ios"], "Windows": ["Windows-mono"], "macOS": None, "Linux": ["Linux"]},
-  "2017.4.40f1": {"Default": ["Unity"], "Android": ["Android"], "iOS": ["Ios"], "Windows": ["Windows"], "macOS": None, "Linux": ["Linux"]}
+  "2020.3.21f1": {"Default": ["Unity"], "Android": ["Android"], "iOS": ["Ios"], "Windows": ["windows-mono"], "macOS": None, "Linux": ["linux-il2cpp"]},
+  "2019.4.32f1": {"Default": ["Unity"], "Android": ["Android"], "iOS": ["Ios"], "Windows": ["windows-il2cpp"], "macOS": None, "Linux": ["linux-il2cpp"]},
```

Comment:
How to select between "il2cpp" and "mono"

Reply:
It really depends on what Unity provides, interestingly, for these two versions on windows there's only windows-il2cpp. It's also possible it will install windows-mono if we sent None here.

Reply:
Got it. What if Unity provides both? Do we have a preference?

Reply:
After some research in
https://forum.unity.com/threads/mono-vs-il2cpp.533416/
https://learn.unity.com/tutorial/memory-management-in-unity?signup=true#5c7f8528edbc2a002053b59b
https://docs.unity3d.com/Manual/IL2CPP.html

Seems il2cpp is the future and are preferred by Unity, so I'd suggest to prefer il2cpp.